docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -84,50 +78,29 @@ core services, security services, etc. As an example, the top-level
 nodes shown when one views the configuration registry might be as
 follows:
 
--   
-
-    edgex *(root namespace)*
-
-    :   -   core *(edgex core services)*
-        -   devices *(device services)*
-        -   security *(security services)*
+- edgex *(root namespace)*
+    -   core *(edgex core services)*
+    -   devices *(device services)*
+    -   security *(security services)*
 
 #### Versioning
 
 Incorporating versioning into the configuration hierarchy looks like
 this.
 
--   
-
-    edgex *(root namespace)*
-
-    :   -   
-
-            core *(edgex core services)*
-
-            :   -   
-
-                    1.0
-
-                    :   -   edgex-core-command
-                        -   edgex-core-data
-                        -   edgex-core-metadata
-
-                -   2.0
-
-        -   
-
-            devices *(device services)*
-
-            :   -   
-
-                    1.0
-
-                    :   -   mqtt-c
-                        -   mqtt-go
-                        -   modbus-go
-
-                -   2.0
+- edgex *(root namespace)*
+    -   core *(edgex core services)*
+        -   1.0
+            -   edgex-core-command
+            -   edgex-core-data
+            -   edgex-core-metadata
+        -   2.0
+    - devices *(device services)*
+        -   1.0
+            -   mqtt-c
+            -   mqtt-go
+            -   modbus-go
+        -   2.0
 
 The versions shown correspond to major versions of the given services.
 These are not necessarily equated with long term support (LTS) releases.
```

Comment:
I recommend we take this line out.  While technically correct, because we don't have an LTS yet (and won't for the foreseeable future), I think this could be confusing to the user.  I don't think it really adds anything to this doc.

Reply:
Done